provider/terraform/custom_code.rb

```diff
@@ -104,6 +104,10 @@ class CustomCode < Api::Object
       # is useful for parsing attributes that are necessary for
       # the Read() method to succeed.
       attr_reader :post_import
+      # This code is run in the generated test file to check that the
+      # resource was successfully deleted. Use this if the API responds
+      # with a success HTTP code for deleted resources
+      attr_reader :delete_check
```

Comment:
```suggestion
      attr_reader :check_destroy
```

Or `test_check_destroy`, to match their `testAccCheck{{resource}}Destroy` naming and disambiguate this from the delete CRUD method.